main.js

```diff
@@ -0,0 +1,14 @@
+
+var $seconds = document.querySelector('#seconds')
+var $start = document.querySelector('#start')
+
+function secondsperseconds() {
+  setInterval(seconds, 1000)
+}
+
+function seconds() {
+  console.log($seconds.textContent)
```

Comment:
When a feature ships, be sure to remove all console logs, except is special circumstances.